manifests/init.pp

```diff
@@ -56,7 +56,9 @@
   $package_name   = $kafka::params::package_name,
   $package_ensure = $kafka::params::package_ensure,
   $group_id       = $kafka::params::group_id,
+  $system_group   = $kafka::params::system_group,
   $user_id        = $kafka::params::user_id,
+  $system_user    = $kafka::params::system_user,
```

Comment:
Yes I could! And I would, if I would be at a point where I can make the switch to puppet4 :sweat:
That's the reason I made this PR to the `puppet3` branch. :sweat_smile: